pixeltable/exec/cache_prefetch_node.py

```diff
@@ -99,7 +99,7 @@ def _fetch_url(self, row: exprs.DataRow, slot_idx: int) -> Optional[str]:
                     if self.boto_client is None:
                         self.boto_client = get_client()
                     self.boto_client.download_file(parsed.netloc, parsed.path.lstrip('/'), str(tmp_path))
-            elif parsed.scheme == 'http' or parsed.scheme == 'https':
+            elif parsed.scheme == 'http' or parsed.scheme == 'https' or parsed.scheme == 'data':
```

Comment:
Let's back this out as well. We should never see images stored as strings in the db.